auth/auth.go

```diff
@@ -25,7 +54,10 @@ func Auth(rw http.ResponseWriter, req *http.Request) bool {
 		return false
 	}
 
-	if token.Claims["hostUuid"] != config.Config.HostUuid {
+	if token.Claims["reportedUuid"] != nil && token.Claims["reportedUuid"] != config.Config.HostUuid {
+		return false
+	}
+	if token.Claims["hostUuid"] != nil && token.Claims["hostUuid"] != config.Config.HostUuid {
```

Comment:
Why was this changed from the old condition?

```go
if token.Claims["hostUuid"] != config.Config.HostUuid {
        return false
}
```

This new code, if the hostUuid is not passed, you don't validate it.  The hostUuid is required and essential for authorization.